src/dataset/dataset.py

```diff
@@ -102,9 +102,9 @@ def __init__(
         rnd: random.Random,
         labelled_pairs: List[Tuple[int, int, int]],
         labelled_val_pairs: List[Tuple[int, int, int]] = None,
-        val_ratio: Optional[float] = 0.2,
+        val_ratio: Optional[float] = 0.1,
         labelled_test_pairs: List[Tuple[int, int, int]] = None,
-        test_ratio: Optional[float] = 0.1,
+        test_ratio: Optional[float] = 0.7,
```

Comment:
Why was the val and test ratio default different than in OpenEA? I changed them. I hope this is fine.

Reply:
This must have happened on accident by a test...